plone/app/textfield/utils.py

```diff
@@ -59,3 +60,38 @@ def getAllowedContentTypes():
         allowed_types = allowed - forbidden
 
     return allowed_types
+
+
+def getDefaultWysiwygEditor():
+    registry = getUtility(IRegistry)
+    try:
+        records = registry.forInterface(IEditingSchema, check=False,
+                                        prefix='plone')
+        default_editor = records.default_editor.lower()
+    except AttributeError:
+        default_editor = 'tinymce'
+    return default_editor
+
+
+def getAvailableWysiwygEditors():
+    registry = getUtility(IRegistry)
+    try:
+        records = registry.forInterface(IEditingSchema, check=False,
+                                        prefix='plone')
+        available = records.available_editors
+    except AttributeError:
+        available = ['TinyMCE']
+    return available
+
+
+def getWysiwygEditor(member_editor, available_editors, default_editor):
```

Comment:
It would be nice if you only had to pass `member_editor` and not the other arguments. But I can see that the current code is easier to test on a low level, which is good.

Suggestion: allow `available_editors` and `default_editor` to be None. In that case, get them from the functions above.